messages/src/main/java/life/genny/messages/process/MessageProcessor.java

```diff
@@ -307,7 +299,7 @@ private HashMap<String, Object> createBaseEntityContextMap(QMessageGennyMSG mess
                     log.info("Fetching contextCodeArray :: " + Arrays.toString(codeArr));
                     // Convert to BEs
                     BaseEntity[] beArray = Arrays.stream(codeArr)
-                            .map(itemCode -> beUtils.getBaseEntity(itemCode))
+                            .map(itemCode -> beUtils.getBaseEntity(itemCode, true))
```

Comment:
This will slow things down. Don't we want to fetch individual attributes for the baseentities?

Reply:
Addressed